firmware/Core/Src/antenna_deploy_drivers/ant_internal_drivers.c

```diff
@@ -1,51 +1,92 @@
 #include "main.h"
 
 #include "stm32l4xx_hal_i2c.h"
+#include "stm32l4xx_hal_def.h"
 #include "antenna_deploy_drivers/ant_internal_drivers.h"
 #include "debug_tools/debug_i2c.h"
-
+#include "debug_tools/debug_uart.h"
+#include "log/log.h"
 #include <stdint.h>
 #include <string.h>
 
-const uint16_t ANT_ADDR = 0x31 << 1;
+/**
+ * @brief This file contains commands for communicating with the antenna deploy system(ADS). The ADS has two microcontrollers which control
+ * the deployment of the antennas. It also has two seperate i2c buses which can be used to communicate with the ADS. The combination of 
+ * microcontroller (mcu) to communicate to and i2c line to use is specified using a value from the Ant_i2c_bus_mcu enum. 
+ * 
+ */
+ 
+
+const uint16_t ANT_ADDR_A = 0x31 << 1; // i2c address of mcu A on ant deploy system
+const uint16_t ANT_ADDR_B = 0x32 << 1; // i2c address of mcu B on ant deploy system
+
 const uint8_t timeout = 50;
```

Comment:
What units is this? Perhaps also include a comment with info about what the timeout is for (HAL_transmit, HAL_receive? Both?)